url_launcher/plugin.go

```diff
@@ -20,34 +20,65 @@ func (p *UrlLauncherPlugin) InitPlugin(messenger plugin.BinaryMessenger) error {
 	channel := plugin.NewMethodChannel(messenger, channelName, plugin.StandardMethodCodec{})
 	channel.HandleFunc("launch", p.launch)
 	channel.HandleFunc("canLaunch", p.canLaunch)
+
+	// Ignored: The plugins doesn't handle WebView.
+	// This call will not do anything, because there is no WebView to close.
+	channel.HandleFunc("closeWebView", func(_ interface{}) (interface{}, error) { return nil, nil })
 	return nil
 }
 
 func (p *UrlLauncherPlugin) launch(arguments interface{}) (reply interface{}, err error) {
-	fmt.Println("launch")
 	var url string
+	var useWebView bool
+	var useSafariVC bool
+	var enableJavaScript bool
+	var enableDomStorage bool
+	var universalLinksOnly bool
 
 	argsMap := arguments.(map[interface{}]interface{})
 	url = argsMap["url"].(string)
 	if url == "" {
 		return nil, errors.New("url is empty")
 	}
+
+	useWebView = argsMap["useWebView"].(bool)
+	if useWebView == true {
+		fmt.Println("plugins.flutter.io/url_launcher: WebView aren't supported on desktop.")
```

Comment:
I think we should prefix these warning lines with 'our' domain; `go-flutter-desktop/plugins/url_lquncher: ....`

What do you think @Drakirus?

Reply:
That a good idea

Reply:
done